docs/reference/generated/kubeadm.md

```diff
@@ -67,7 +67,7 @@ following steps:
    See [Securing your installation](#securing-more) for hardening.
 
 1. kubeadm installs add-on components via the API server.  Right now this is
-   the internal DNS server and the kube-proxy DaemonSet.
+   the internal DNS server and the kube-proxy DaemonSet. There is no requirement for specifying kube-proxy bind address in kubeadm.conf. The kube-proxy bind address is dynamically assigned based on the address type of API server (i.e. v4 will assign 0.0.0.0 & v6 will assign :: ).
```

Comment:
replace `kubeadm.conf` with `the kubeadm configuration file`

Reply:
> based on the address type of API server

do you mean the `--bind-address` of the API server? If so, can we explicit reference that

Reply:
The proxy address is determined based on the address type (IPv4 or IPv6) of the
`--apiserver-advertise-address`. However, this should not be mentioned b/c the kubeadm conf file is not used for managing the proxy config.